includes/wc-update-functions.php

```diff
@@ -1993,6 +1993,7 @@ function wc_update_370_tax_rate_classes() {
 			WC_Tax::create_tax_class( $class );
 		}
 	}
+	delete_option( 'woocommerce_tax_classes' );
```

Comment:
Good call on cleaning this up. 👍

Reply:
Added backward compatible code to the function. Just noting that this used to be called on a filter on wp_options, but non the less I added the functionality to transition to the new table if this happens to get called via an option save or manually since this is a public status function.